<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addCar</class>
 <widget class="QDialog" name="addCar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="addCarButton">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>280</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monolisa Trial</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:14px;
font-weight:bold;
border-radius:10px;
background-color:green;
color:white;
font-family:&quot;Monolisa Trial&quot;;</string>
   </property>
   <property name="text">
    <string>Add Car</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>430</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monolisa Trial</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:14px;
font-weight:medium;
border-radius:10px;
background-color:white;
color:black;
font-family:&quot;Monolisa Trial&quot;;</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bookCarButton">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>330</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monolisa Trial</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:14px;
font-weight:medium;
border-radius:10px;
background-color:orange;
color:black;
font-family:&quot;Monolisa Trial&quot;;</string>
   </property>
   <property name="text">
    <string>Book Car</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-220</x>
     <y>-10</y>
     <width>911</width>
     <height>381</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../car-bg.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="mainMenuTitle">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>180</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MonoLisa Trial</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;
font-weight:bold;</string>
   </property>
   <property name="text">
    <string>Main Menu</string>
   </property>
  </widget>
  <widget class="QLabel" name="addCarLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>211</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MonoLisa Trial</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:orange;
font-weight:bold;</string>
   </property>
   <property name="text">
    <string>Add Car</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="brandLineEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>120</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MonoLisa Trial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
border-radius:10px;
border:2px solid white;
color:white;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="carNoLineEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MonoLisa Trial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
border-radius:10px;
border:2px solid white;
color:white;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="addCarButtonInAddCar">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>300</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monolisa Trial</family>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:14px;
font-weight:bold;
border-radius:10px;
background-color:green;
color:white;
font-family:&quot;Monolisa Trial&quot;;</string>
   </property>
   <property name="text">
    <string>Add Car</string>
   </property>
  </widget>
  <widget class="QLabel" name="carNoLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MonoLisa Trial</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;
font-weight:bold;</string>
   </property>
   <property name="text">
    <string>Car No.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitAddCarButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>300</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monolisa Trial</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:14px;
font-weight:medium;
border-radius:10px;
background-color:white;
color:black;
font-family:&quot;Monolisa Trial&quot;;</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="brandLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MonoLisa Trial</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;
font-weight:bold;</string>
   </property>
   <property name="text">
    <string>Brand</string>
   </property>
  </widget>
  <widget class="QLabel" name="carTypeLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MonoLisa Trial</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;
font-weight:bold;</string>
   </property>
   <property name="text">
    <string>Car Type</string>
   </property>
  </widget>
  <widget class="QListWidget" name="carTypeWidgetList">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>170</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MonoLisa Trial</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
border-radius:10px;
border:2px solid white;
color:white;</string>
   </property>
  </widget>
  <widget class="QLabel" name="fareLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MonoLisa Trial</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;
font-weight:bold;</string>
   </property>
   <property name="text">
    <string>Fare</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fareLineEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>230</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MonoLisa Trial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
border-radius:10px;
border:2px solid white;
color:white;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
